extensions/objects/templates/set_of_html_string_editor.html

```diff
@@ -1,3 +1,13 @@
+<style>
+md-checkbox .md-icon {
```

Comment:
Indent this and the lines inside the style tag by 2.

Also, this has the same issue as some other PRs: any CSS you write here applies to the whole page. Please scope it properly.

Reply:
Hi @seanlip 
I'm a bit confused here. Do you mean to convert this css into a nested form like 
``` 
.md-checkbox {
   .md-icon {
      top: 2px;
  }
  .md-label {
      vertical-align: top; 
 }
}
``` 
Or you want me to add an extra class and then change CSS to effect that particular class only  ?

Reply:
The indentation should be:

```
<style>
  selector-1 {
     ...
  }
</style>
```

For the scoping, you may need to do something similar to the audio-bar thing. Basically, any solution that ensures that md-checkboxes outside this template do not get affected (you should test this).

Reply:
Will adding the ``container-fluid`` class be the right solution?

Reply:
Probably not. You need to do something that is guaranteed to be unique to this template.

Reply:
So, would it be fine to add new class like..
```
 <div class="container-fluid string-set-editor">
```

Reply:
Take a look at the HTML on which this template is placed. Is this a custom tag of the form `<set-of-html-string-editor></set-of-html-string-editor>`?

If so, then you can prepend your selectors with `set-of-html-string-editor` to scope the CSS changes to just this element. If not, then you can add a new class, but be sure to name it appropriately. For example, you might just use a class that's called "set-of-html-string-editor" to avoid ambiguity.

Reply:
Thanks! I got it :smile:

Reply:
Thank you @seanlip for the detailed explanation, it gave me better insight into all the changes.